account_billing/views/account_billing.xml

```diff
@@ -0,0 +1,162 @@
+                <filter name="state_billed" string="Billed" domain="[('state','=','billed')]" help="Billed Billing"/>
+                <separator/>
+                <filter string="Vendor" domain="[('bill_type','=','in_invoice')]" name="vendor_filter"/>
+                <filter string="Customer" domain="[('bill_type','=','out_invoice')]" name="customer_filter"/>
+                <group expand="0" string="Group By">
+                    <filter name="group_by_customer" string="Customer" context="{'group_by':'partner_id'}"/>
+                    <filter name="group_by_state" string="Status" context="{'group_by':'state'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="view_account_billing_form" model="ir.ui.view">
+        <field name="name">account.billing.form</field>
+        <field name="model">account.billing</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button name="validate_billing" class="oe_highlight" states="draft" string="Validate" type="object"/>
+                    <button name="action_cancel" states="draft" string="Cancel" type="object"/>
+                    <button name="action_cancel" states="billed" string="Cancel" type="object" confirm="Are you sure to cancel this document?"/>
+                    <button name="action_cancel_draft" states="cancel" string="Set to Draft" type="object"/>
+                    <field name="state" widget="statusbar" statusbar_visible="draft,billed"/>
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button class="oe_stat_button"
+                                name="invoice_relate_billing_tree_view"
+                                type="object"
+                                attrs="{'invisible':[('state', 'not in', ['billed'])]}"
+                                icon="fa-pencil-square-o">
+                            <field name="invoice_related_count" string="Invoices" widget="statinfo"/>
+                        </button>
+                    </div>
+                    <h1>
+                        <label for="name" string="Draft Billing" attrs="{'invisible': [('state','!=', 'draft')]}"/>
+                        <field name="name" readonly="1" class="oe_inline" attrs="{'invisible': [('state', '=', 'draft')]}"/>
+                    </h1>
+                    <group>
+                        <group>
+                            <field name="partner_id" attrs="{'readonly': [('state', '!=', 'draft')]}" context="{'default_supplier': bill_type == 'in_invoice', 'default_customer': bill_type == 'out_invoice'}"/>
+                            <field name="bill_type" invisible="1"/>
+                        </group>
+                        <group>
+                            <field name="billing_date_type"/>
```

Comment:
Would it make more sense to move the field below `date` so that `billing_date_type` and `threshold_date` are next to each other?

Reply:
I think I'll move date below threshold_date because ```billing_date_type``` ```threshold_date``` ```date``` beautiful than ```date``` ```billing_date_type``` ```threshold_date```

Reply:
@yostashiro now I change string **Invoice Date Threshold** and **Due Date Threshold** to **Threshold Date** and move field date below threshold_date
I got this:
![Selection_014](https://user-images.githubusercontent.com/52144935/70407173-d6797000-1a75-11ea-917e-4d1d70bad71d.png)
for me it's look very weird because Billing Date is below Threshold Date.
**It's should be**
   - Billing Date Type:
   Billing Date:
   Threshold Date:

What do you think?